catalogue/webapp/components/Calendar/CalendarStyles.tsx

```diff
@@ -51,13 +47,32 @@ export const Td = styled.td`
   }
 
   &[tabindex='0']:focus {
-    outline: none;
     background: ${props => props.theme.color('green')};
     color: ${props => props.theme.color('white')};
+    outline: 0;
+    .is-keyboard & {
```

Comment:
I think there's a `useKeyboard` hook that gives us a bit more React-y way of doing this, passing a prop to the styled component. Although I think we can/should move to using `focus-visible` for this stuff anyhow. So maybe not worth updating now.